src/components/app/Main.tsx

```diff
@@ -1,6 +1,6 @@
 export default function Main({children}) {
   return (
-    <main className="max-w-screen-lg w-full mx-auto sm:p-8 p-5 flex flex-col flex-grow">
+    <main className="max-w-screen-lg w-full mx-auto sm:p-8 p-5 flex flex-col flex-grow prose">
```

Comment:
was trying to figure out how to format the mdx above so I added the `prose` here, but I don't think this is a great way to do that and I'm wondering how better to manage/handle mdx 🤔